packages/vsphere-cpi/charts/values.yaml

```diff
@@ -0,0 +1,23 @@
+# Default values for vsphere-cpi.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  cattle:
+    systemDefaultRegistry: ""
+
+image:
+  repository: gcr.io/cloud-provider-vsphere/cpi/release/manager
+  tag: v1.2.1
```

Comment:
@PennyScissors which image should be replaced here? Can you also reference the image-mirror PR in this comment?

Reply:
Yes it needs to be in format: `rancher/mirrored-`.....

Reply:
https://github.com/rancher/image-mirror/pull/78 will track creating this image. Updating this PR to use the newly created repo name and tag

Reply:
https://hub.docker.com/r/rancher/mirrored-cloud-provider-vsphere-cpi-release-manager/tags?page=1&ordering=last_updated exists